floyd/cli/data_upload_utils.py

```diff
@@ -56,17 +60,22 @@ def initialize_new_upload(data_config, access_token):
 
     creds = DataClient().new_tus_credentials(data_id)
     if not creds:
+        # TODO: clean up
         sys.exit(1)
 
-    data_endpoint = TusDataClient().initialize_upload(tarball_path,
-                                                      metadata={"filename": data_id},
-                                                      auth=creds)
+    data_resource_id = creds[0]
```

Comment:
Is there a chance that this will sometimes be different than the data_id? (I assume it's possible or we wouldn't be using data_resource_id) If so, it could cause problems with auth on the tus server.